src/tests/gssapi/t_gssapi.py

```diff
@@ -143,22 +143,23 @@
 realm.run(['./t_inq_cred', '-k', '-b'], expected_msg=realm.host_princ)
 
 # Test gss_export_name behavior.
-out = realm.run(['./t_export_name', 'u:x'])
-if out != '0401000B06092A864886F7120102020000000D78404B5242544553542E434F4D\n':
+output = realm.run(['./t_export_name', 'u:x'])
```

Comment:
Can we use expected_msg for these?

(Using "output" as a variable name for the output of realm.run() is a historical botch, since there is a k5test.py function named "output".  So standardizing on "output" for these tests is worse than standardizing on "out", but using expected_msg would render the issue moot.)